source/core/Math.cpp

```diff
@@ -366,11 +364,15 @@ DECLARE_SCALE2X_INTN_HELPER(Single)
 #define DECLARE_VIREO_COMPARISON_PRIMITIVES(TYPE) \
     DECLARE_VIREO_PRIMITIVE2(IsLT0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) <  0)) \
     DECLARE_VIREO_PRIMITIVE2(IsGE0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) >= 0)) \
+    DECLARE_VIREO_PRIMITIVE2(IsLE0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) <= 0)) \
+    DECLARE_VIREO_PRIMITIVE2(IsGT0##TYPE, TYPE, Boolean, (_Param(1) = _Param(0) >  0)) \
     DECLARE_VIREO_COMPARISON_PRIMITIVES_BASE(TYPE)
 
 #define DECLARE_VIREO_COMPARISON_PRIMITIVES_BOOLEAN() \
     DECLARE_VIREO_PRIMITIVE2(IsLT0Boolean, Boolean, Boolean, (_Param(1) = false)) \
     DECLARE_VIREO_PRIMITIVE2(IsGE0Boolean, Boolean, Boolean, (_Param(1) = true)) \
+    DECLARE_VIREO_PRIMITIVE2(IsLE0Boolean, Boolean, Boolean, (_Param(1) = (_Param(0) ? 1 : 0) <= 0)) \
+    DECLARE_VIREO_PRIMITIVE2(IsGT0Boolean, Boolean, Boolean, (_Param(1) = (_Param(0) ? 1 : 0) >  0)) \
```

Comment:
These four (LT0....GT0) with boolean input appear to be semantic analysis errors in LV. So, we may not get here for LV generated VIA.

Reply:
I attempted to remove the Macros, but I got errors. It appears another code is expecting the functions to exists. I'll look into removing the Macros in another PRs.